### ISSUE TITLE
Crashes while downloading torrent

### ISSUE THREAD

**Issue by raeesiqbal** [tatsuhiro-t/aria2#610]
The program crashes after a few moments of starting the download. The behavior is same with all torrent files but some magnet links never start downloading.

Torrent file: [Sintel](https://webtorrent.io/torrents/sintel.torrent)
Version: `aria2 1.21.0`
Platform: Windows 10 x64
Command: `aria2c --file-allocation=none "https://webtorrent.io/torrents/sintel.torrent"`

**Comment by tatsuhiro-t** [comment]
Crash has been confirmed.  Thank you for reporting this.

**Comment by tatsuhiro-t** [comment]
It looks like the crash was caused by gmp bug: https://gmplib.org/#STATUS

```
Issues with GMP 6.1.0:

    An assembly file which is used for Intel Broadwell and Intel Skylake (except crippled Pentiums and Celerons without BMI2) will not work correctly for Windoze. Patch.
```
Building gmp with the suggested patch fixes this issue.

**Comment by tatsuhiro-t** [comment]
Rebuilt aria2 with patched gmp, which is available as aria2-1.21.0-win-64bit-build2.zip under https://github.com/tatsuhiro-t/aria2/releases/tag/release-1.21.0
Could you confirm that this fixes this issue?

**Comment by raeesiqbal** [comment]
Fix confirmed with both magnet links and torrent files. Much thanks, you're awesome :)
